src/streams/EditStreamCard.js

```diff
@@ -68,20 +63,21 @@ export default class EditStreamCard extends PureComponent<Props, State> {
   render() {
     const { initialValues, streamId } = this.props;
     const { name } = this.state;
+    const { styles } = this.context;
 
     return (
       <View>
         <Label text="Name" />
         <Input
-          style={styles.marginBottom}
+          style={styles.halfMarginBottom}
```

Comment:
Even though `halfMarginBottom` which is `8` is closer to the older value of `10`, you should use the new utility one of `marginBottom` that is `16`.

Reply:
using `marginBottom` causes the view to become scrollable (it doesn't seem to expand in height) and will require further changes to correct that.

Although that is probably be a good thing since that won't become an issue if anything is added to this screen later that causes it to exceed the current view height.
I will the changes required and update the PR.